pkg/packer/manifests/vsphere/base/linux/centos/http/8/ks.cfg

```diff
@@ -54,13 +52,22 @@ python3
 # Enable/disable the following services
 services --enabled=sshd
 
-%post --nochroot --logfile=/mnt/sysimage/root/ks-post.log
+%post --nochroot --logfile=/mnt/sysimage/var/log/ks-post.log
 # Disable quiet boot and splash screen
 sed --follow-symlinks -i "s/ rhgb quiet//" /mnt/sysimage/etc/default/grub
 sed --follow-symlinks -i "s/ rhgb quiet//" /mnt/sysimage/boot/grub2/grubenv
 
 # Passwordless sudo for the user 'builder'
 echo "builder ALL=(ALL) NOPASSWD: ALL" >> /mnt/sysimage/etc/sudoers.d/builder
+
+# copy ssh authorized file
+echo "mount cdrom"
+mkdir -p /mnt/sysimage/cdrom
+mkdir -p /mnt/sysimage/home/builder/.ssh
+mount /dev/sr1 /mnt/sysimage/cdrom
+echo "copy authorized keys"
+cp /mnt/sysimage/cdrom/authorized_keys /mnt/sysimage/home/builder/.ssh/authorized_keys
+umount /dev/sr1
```

Comment:
Don't really like this part as the KS part should be templated by using `http_content` in the packer builder.

Instead
```
user --name=${var.ssh_username} --groups=${var.ssh_username},wheel
sshkey --username=${var.ssh_username} "${data.sshkey.install.public_key}"
```

would be my suggestion...